src/components/Utils/queries.js

```diff
@@ -161,15 +161,15 @@ export const queryGuppyForRawDataAndTotalCounts = (
   offset = 0,
   size = 20,
   accessibility = 'all',
-  format = 'json',
+  format,
 ) => {
-  let queryLine = 'query ($format: Format) {';
-  if (gqlFilter || sort) {
-    queryLine = `query ($format: Format, ${sort ? '$sort: JSON,' : ''}${gqlFilter ? '$filter: JSON,' : ''}) {`;
+  let queryLine = 'query {';
+  if (gqlFilter || sort || format) {
```

Comment:
This works for the undefined case, but are we okay with populating an error if the format passed is _not_ supported from the Format schema {tsv, csv, json}?

Reply:
Yeah if the format vaule is not supported then Guppy will returns 400 and that will causing the front-end part to throw out an error. And I think that is okay for this case. The TSV/CSV conversion only works for the buttons anyways. It is not something that user can type in from the `Query` page and get a TSV/CSV. So the only place to mess it up is someone configured a button with something like `data-abc`.

Reply:
I can also add more logic into `GuppyWrapper/index.jsx` to print an warning in console if the input `format` is not among `FILE_FORMATS` to polish it up

Reply:
Yeah that might be helpful to developers if a button was not configured correctly (or file type not supported) and it will still throw the Guppy error which also shows the response from graphql.